.github/workflows/php.yml

```diff
@@ -164,29 +164,31 @@ jobs:
       - name: Codecov
         uses: codecov/codecov-action@v1
 
-  infection:
-    name: "3️⃣ PHP: Mutation testing"
-    runs-on: "ubuntu-latest"
-    needs:
-    - "tests"
-    steps:
-      - name: Checkout code
-        uses: actions/checkout@v3
-
-      - name: Set Up Imagick, FFmpeg & Exiftools
-        run: |
-          sudo apt-get update
-          sudo apt-get --fix-broken install
-          sudo apt-get -y install ffmpeg libimage-exiftool-perl libmagickwand-dev
-
-      - name: Setup PHP
-        uses: shivammathur/setup-php@2.15.0
-        with:
-          php-version: "latest"
-          coverage: xdebug
-
-      - name: Install Composer dependencies
-        uses: "ramsey/composer-install@v2"
-
-      - name: Run Infection
-        run: vendor/bin/infection
+  # support for mutation testing.
+  #
+  # infection:
+  #   name: "3️⃣ PHP: Mutation testing"
+  #   runs-on: "ubuntu-latest"
+  #   needs:
+  #   - "tests"
+  #   steps:
+  #     - name: Checkout code
+  #       uses: actions/checkout@v3
+
+  #     - name: Set Up Imagick, FFmpeg & Exiftools
+  #       run: |
+  #         sudo apt-get update
+  #         sudo apt-get --fix-broken install
+  #         sudo apt-get -y install ffmpeg libimage-exiftool-perl libmagickwand-dev
+
+  #     - name: Setup PHP
+  #       uses: shivammathur/setup-php@2.15.0
+  #       with:
+  #         php-version: "latest"
+  #         coverage: xdebug
+
+  #     - name: Install Composer dependencies
+  #       uses: "ramsey/composer-install@v2"
+
+  #     - name: Run Infection
+  #       run: vendor/bin/infection
```

Comment:
I don't like code which is commented out, because nobody will remember why this code has been commented out. Maybe we can simply removed it?

And what is about this this strange new errors about allegedly added code lines (which do not seem to be added actually) and are reported as not covered by tests?!

Reply:
This is mutation testing.
I tries to change the code by e.g. replacing `<=` by `<` and see if the test still passes, if the tests don't fail on the mutant then it is reported as an escaped mutant.